Gemfile

```diff
@@ -27,6 +27,7 @@ gem "puppet", :path => File.dirname(__FILE__), :require => false
 gem "facter", *location_for(ENV['FACTER_LOCATION'] || ['> 2.0', '< 4'])
 gem "hiera", *location_for(ENV['HIERA_LOCATION'] || ['> 1.0', '< 3'])
 gem "rake", "10.1.1", :require => false
+gem "aws-sdk-core", "2.0.5"
```

Comment:
Just following up a comment from the triage video on the 4th of March.  I didn't mean to imply that the aws-sdk-core gem should be built into the standard puppet suite.  I only added it here to signal that it was required to have my code work.

I agree it needs to be moved into a development group.

In my view those users of puppet who wish to use the amazon cloud should install the gem if required.